### ISSUE TITLE
Apple M4 - Mac UTM open to black screen

### ISSUE THREAD

**Issue by DuaneMIS** [utmapp/UTM#6794]
Black Screen upon UTM Build.
When trying to spin up a NEW Mac Monterey UTM, I can see the percentage indicator loading up to and then at 100% the screen changes from the rolling circular lines to a black screen and seems to halt. The last attempt sat on a black screen overnight for 6 hours.

The load process was to use the VM Wizzard, point to the downloaded UniversalMac_12.4_21F79_Restore.ipsw file that I downloaded from IPSW.me. - I have had the same result with different IPSW files as well.

**Configuration**  
* UTM Version: 4.5.4 from Apple App Store
* macOS Version:  Sequoia 15.1
* Mac Chip (Intel, M1, ...): M4 Max Pro, 48gb Ram (this is a new machine from Apple)

**Crash log**  
I do not see any crash logs related to UTM.

Here is the link to the VM:
https://drive.google.com/file/d/1VPyPIuanWUFKWHmti8DsPtQxnO1jW5jq/view?usp=share_link

Diagnostic Report
[com.apple.Virtualization.VirtualMachine_2024-11-10-215355_MacBook-Pro.cpu_resource.diag.zip](https://github.com/user-attachments/files/17703793/com.apple.Virtualization.VirtualMachine_2024-11-10-215355_MacBook-Pro.cpu_resource.diag.zip)

**Debug log**  
Mac System Log
[system.log.zip](https://github.com/user-attachments/files/17703839/system.log.zip)

**Upload VM**  
[config.plist.zip](https://github.com/user-attachments/files/17703915/config.plist.zip)

**Comment by theevilbit** [comment]
I have the same issue on an M4 Max.

**Comment by theevilbit** [comment]
I also can't run Monterey VM on Virtualbuddy. So this might be a M4/Apple issue

**Comment by theevilbit** [comment]
Early version of Ventura seems to have issues as well.

**Comment by theevilbit** [comment]
After further testing macOS 13.4 is the oldest version I can virtualize on my M4 Max.

**Comment by osy** [comment]
I have the same issue with my M4 Pro. I think this is an Apple bug because the same VM works fine on 15.1 on an M1 mac

**Comment by theevilbit** [comment]
Agree, the same is going on with all virtualization software. I suggest everyone to submit a Feedback, so maybe this will get fixed. I hope this won't be a software-hardware incompatibility issue which won't get addressed.

**Comment by DuaneMIS** [comment]
> Agree, the same is going on with all virtualization software. I suggest everyone to submit a Feedback, so maybe this will get fixed. I hope this won't be a software-hardware incompatibility issue which won't get addressed.

Where should we submit the 'Feedback' here or at Apple? Sorry new here and trying to find my way around.

**Comment by theevilbit** [comment]
opened a thread at Apple: https://developer.apple.com/forums/thread/768674

**Comment by ideologysec** [comment]
It has at least been acknowledged per the thread linked in the Apple Developer forums, as a bug in need of fixing.

https://developer.apple.com/documentation/macos-release-notes/macos-15_2-release-notes#Virtualization

**Comment by theevilbit** [comment]
It has been fixed in 15.2.

**Comment by theevilbit** [comment]
macOS 15.2

**Comment by falkolab** [comment]
macOS 15.3 same issue
UTM 4.6.4 (107)

**Comment by ideologysec** [comment]
> macOS 15.3 same issue UTM 4.6.4 (107)

I can't reproduce that here. Can you provide more details?

<img width="644" alt="Image" src="https://github.com/user-attachments/assets/426a3c53-c7f2-4a6b-acde-16077c57af59" />

**Comment by falkolab** [comment]
> > macOS 15.3 same issue UTM 4.6.4 (107)
> 
> I can't reproduce that here. Can you provide more details?
> 
I found the solution by unchecking Display -> Retina mode

**Comment by FlyingCodeMonkey** [comment]
I have an M2 Pro and this issue has just started happening to me. I'm running macOS 15.5, UTM 4.6.5 (108) downloaded from the web. I have a variety of VMs with different OSes installed on them (macOS 13, 14, 15 and 26), and I can't start any of them. Up until now, they've all been running fine, but now they all just start to a black screen.

I've tried enabling and disabling HiDPI and Dynamic Resolution in the Display settings but it doesn't help.

Can anyone help please.

**Comment by Tushar0744** [comment]
@FlyingCodeMonkey  your issue got resolved, I am facing the similar issue on Apple M4 with Sequoia 15.3.1, here i was having Sequoia, Sonoma and Tahoe VMs on Sequoia 15.3.1, sequoia VM is working fine but other two Sonoma and Tahoe are giving black screen issue, tried out various things but still not able to resolve.
Do you have any idea about it?

**Comment by FlyingCodeMonkey** [comment]
@Tushar0744 I'm afraid I can't remember exactly what the solution was, but I think it was related to the VM being on an external volume. When I moved it back over to the internal HD, it started up fine. Other than that, I'm not sure. Sorry!
